packages/uploadthing/src/sdk/utils.ts

```diff
@@ -83,6 +85,7 @@ export const uploadFilesInternal = async (
         formData.append(key, value);
       });
       formData.append("file", file);
+      formData.set("Policy", "");
```

Comment:
? @juliusmarminge

Reply:
Ohhh shoot that was my way of testing a failed upload hahhh. Sorry about thst. Revert